### DISCUSSION TITLE
Install i2p on router hardware?

### DISCUSSION THREAD

**Discussion by ghost** [PurpleI2P/i2pd#1932] [Q&A]
**Can you install i2p on OpenWrt?**

**Comment by r4sas** [comment]
I2P - no. But i2pd is already available on OpenWRT repository since 18.x versions.

**Comment by ghost** [comment]
@r4sas 
**Is there any guides out there?** 

My use case would be to have router with i2p on it similar to VPN/Tor router but with no outproxies. Just so connected devices could browse i2p sites without having to install i2p on each device and to avoid leaks.

I see that someone mentioned the [latest version doesn't have SSU2 enabled](https://github.com/PurpleI2P/i2pd-openwrt/issues/1) in openwrt packages...

**Reply by r4sas** [reply]
Which OpenWRT version are you using?
